client/src/components/ProfileDetails/CreateReview/CreateReview.tsx

```diff
@@ -0,0 +1,90 @@
+import { useState } from 'react';
+import { Avatar, Box, Button, TextField, Typography } from '@material-ui/core';
+import Rating from '@material-ui/lab/Rating';
+
+import useStyles from './useStyles';
+import { useAuth } from '../../../context/useAuthContext';
+import { useSnackBar } from '../../../context/useSnackbarContext';
+import { ReviewsApiData } from '../../../interface/ReviewApiData';
+import { ReviewForm } from '../../../interface/Review';
+import { createReview } from '../../../helpers/APICalls/reviews';
+
+interface Props {
+  sitterId: string;
+}
+
+export default function CreateReview({ sitterId }: Props): JSX.Element {
+  const { loggedInUserDetails, updateReviewsContext } = useAuth();
+  const { updateSnackBarMessage } = useSnackBar();
+  const [toggle, setToggle] = useState<boolean>(false);
+  const [review, setReview] = useState<ReviewForm>({
+    rating: 0,
+    message: '',
+  });
+  const classes = useStyles();
+
+  const handleSubmit = () => {
+    createReview(sitterId, review).then((data: ReviewsApiData) => {
```

Comment:
Check if message or review is empty before sending  API request

Reply:
As messaging and ratings have default values ( ' ' and 0), what is the reason behind the check?

Reply:
So make sure that we don't send empty reviews to backend.